Vizzuality/heco-invest
# LET-327 Asynchronous translation of the project

Integrated Google Translation API to automatically translate columns when project is created. It is handled by startig a background job in the controller, not via model callback.

I haven't  been able to figure out how to parametrise this API without having an env var with GCP project id available.

When running terraform plan I see that the new env var will be added and the new service "created". I honestly don't know if that is the equivalent of clicking the "enable" button.

## Testing instructions

Locally:
- set the project id env var in .env
- start cloudtasker
- create a project using rswag docs
- check in server log that translation succeeded (no idea how to change lng using rswag docs)

## Tracking

https://vizzuality.atlassian.net/browse/LET-327?atlOrigin=eyJpIjoiMTU4YzI1NjFkNGFjNGUyOGFiM2I2YjljYTY4NzljNGUiLCJwIjoiaiJ9
